### ISSUE TITLE
Top App Bar Background Image and transformation

### ISSUE THREAD

**Issue by zaheraldefai** [material-components/material-components-web#4740]
Hi All,

I'm trying out the [Top App Bar](https://github.com/material-components/material-components-web-react/tree/master/packages/top-app-bar) and it works wonderful. But I'm looking for two features that are described in the [Top App Bar Guidelines](https://material.io/design/components/app-bars-top.html):

### Background Image

![mio-design_assets_1UmsSQiiLgULeUnPsyvwdtixhQ7VfqRmN_topappbars-images](https://user-images.githubusercontent.com/12548165/57975730-51424700-79cf-11e9-81de-33e978b4fa3f.png)

I didn't find out how to include a background image unless I change the CSS manually. But in my use case the image is always a different one and I won't like to change the CSS manually. I would like to make it in the same way like in the Card-component with Card-Media, which is the much cleaner way.

### Transformation after scroll

![mio-design_assets_1d3jb571H-QqU8rMvcvksU1e8-Di0lgu0_topappbars-behavior-scrolling-prominent-to-regular](https://user-images.githubusercontent.com/12548165/57975688-476c1400-79ce-11e9-82f4-44591a6d1fe3.gif)

Here is the exact Guideline description: "When scrolling up, prominent top app bars using imagery can transform into normal top app bars. They should not return to prominent mode until the user scrolls back to the top of the page."

<br>

**My questions:**
* Are these points already possible now?
* If no, are there any plans to offer this in the feature and when?
* If no plans are there. Do you know some tips to reach these goals? Should I use this Component without the react wrapper?

Best Regards,
Zaher

**Comment by zaheraldefai** [comment]
Hi @moog16, thanks for the quick reply. Does this mean there are future plans for offering these two features?

I'm asking because if there are plans then we would start with using the existing component and make some smaller css customization, so the migration to the newer version would be easier.

**Comment by moog16** [comment]
I don't think we had a plan to add this to our API, but since it is in the spec we can add it. Are you available to open a PR?

**Comment by zaheraldefai** [comment]
Many thanks for adding it and thank you for offering me this chance to contribute to this project. Unfortunately, I have already too much on my plate and therefore I cannot take the time for this chance.

**Comment by moog16** [comment]
No worries - thanks for letting us know! We'll keep the issue open anyways.
